public/app/story/storyCtrl.js

```diff
@@ -60,6 +60,61 @@ define([], function() {
       $scope.errorMessage = undefined;
     }
 
+    function getUser() {
+      userService.getUser()
+        .success(function (dataUser) {
+          $scope.users = dataUser;
+          
+                
+        })
+        .error(function (error) {
+          $scope.status = 'Unable to load customer data: ' + error.message;
+        });
+    }
+  
+    getUser();
+
+
+  $scope.user =  appConstants.getItem('currentUser');
+
+  $scope.model = {
+   selectedStoryList : []

```

Comment:
why you need a model here?

you can initialize this way $scope.selectedStoryList = [];